AlfrescoApp/App Delegate/AppDelegate.m

```diff
@@ -133,10 +133,10 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     
     [MigrationAssistant runDownloadsMigration];
     
-    if([[RealmSyncManager sharedManager] isCoreDataMigrationNeeded])
-    {
+//    if([[RealmSyncManager sharedManager] isCoreDataMigrationNeeded])
```

Comment:
Is this comment for testing purposes only? If so, would you please add a warning?